test/main.pony

```diff
@@ -70,7 +66,7 @@ class _TestSync is UnitTest
 
 class _TestSelect is UnitTest
   let _ponyc_versions: Array[String] val =
-    ["release-0.33.1"; "release-0.33.0"]
+    ["release-0.33.2"; "release-0.34.0"]
```

Comment:
Do we have to update this for every release?

Reply:
No, this test just requires 2 versions available on all supported platforms

Reply:
So what are the hardcoded versions? I'm confused by those.

Reply:
The test installs both versions, and tests selecting between them using ponyup.

Reply:
Right so if we add FreeBSD support, then we need to change these right? And it impacts on every platform.

Reply:
Yes, and the tests would require two releases (or nightly releases temporarily) to be available.